src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -0,0 +1,150 @@
+import { getGenerator, IntrospectionEngine } from '@prisma/sdk'
+import * as fs from 'fs-jetpack'
+import * as Path from 'path'
+import pkgup from 'pkg-up'
+import Database from 'sqlite-async'
+
+const engineVersion = require('../../../package.json').prisma.version
+
+process.env.SKIP_GENERATE = 'true'
+
+const pkgDir = pkgup.sync() || __dirname
+const engine = new IntrospectionEngine()
+
+beforeAll(() => {
+  fs.remove(getKaseDir(''))
```

Comment:
The usage of `kase` instead of `case` is due to the reserved nature of `case` in JavaScript. Namely on LOC 49 `kase` param would not be valid. To be consistent I've used `kase` term everywhere.

An alternative term would be `scenario` but it is twice as long. Still, if `kase` is confusing we should go with `scenario` then.

Reply:
Kase is a bit like `Käse` and I now read it as get🧀Dir 😄 
It's a bit weird to me and a rename would remove the `you made a typo` thinking too.

Reply:
Ok let's try `scenario`